Resources/Prototypes/_Goobstation/Entities/Mobs/Species/yowie.yml

```
@@ -0,0 +1,56 @@
+- type: entity
+  parent: BaseMobSpeciesOrganic
+  id: BaseMobYowie
+  name: Urist McYowie
+  abstract: true
+  components:
+  - type: Sprite
+    scale: 1.3, 1.3 
```

yamllint

[error] trailing spaces

(trailing-spaces)